src/test/TestGhoCcipSteward.t.sol

```diff
@@ -240,16 +240,186 @@ contract TestGhoCcipSteward is TestGhoBase {
       rate: 0
     });
     vm.prank(RISK_COUNCIL);
-    vm.expectRevert();
+    vm.expectRevert(
+      abi.encodeWithSelector(
+        RateLimiter.InvalidRatelimitRate.selector,
+        RateLimiter.Config({isEnabled: true, capacity: 0, rate: 0})
+      )
+    );
     GHO_CCIP_STEWARD.updateRateLimit(
       remoteChainSelector,
       invalidConfig.isEnabled,
       invalidConfig.capacity,
       invalidConfig.rate,
-      rateLimitConfig.isEnabled,
-      rateLimitConfig.capacity,
-      rateLimitConfig.rate
+      invalidConfig.isEnabled,
+      invalidConfig.capacity,
+      invalidConfig.rate
+    );
+  }
+
+  function testChangeEnabledRateLimit() public {
+    RateLimiter.TokenBucket memory outboundConfig = MockUpgradeableLockReleaseTokenPool(
+      GHO_TOKEN_POOL
+    ).getCurrentOutboundRateLimiterState(remoteChainSelector);
+    RateLimiter.TokenBucket memory inboundConfig = MockUpgradeableLockReleaseTokenPool(
+      GHO_TOKEN_POOL
+    ).getCurrentInboundRateLimiterState(remoteChainSelector);
+
+    // assert both inbound & outbound rate limiters are enabled
+    assertTrue(outboundConfig.isEnabled);
+    assertGt(outboundConfig.capacity, 0);
+    assertGt(outboundConfig.rate, 0);
+
+    assertTrue(inboundConfig.isEnabled);
+    assertGt(inboundConfig.capacity, 0);
+    assertGt(inboundConfig.rate, 0);
+
+    RateLimiter.Config memory disableLimitConfig = RateLimiter.Config({
```

Comment:
add short comment saying capacity and rate must be 0, enforced by the token pool

Reply:
added - [29ccd61](https://github.com/aave/gho-core/pull/427/commits/29ccd61119b52ee02d5106d7f9e43a6fe3ec84ff)